config/bundler_helper.rb

```diff
@@ -5,22 +5,30 @@
 module BundlerHelper
   def self.rails_env
     @rails_env ||= ENV["RAILS_ENV"] ||
-      parse_value_from_file("diaspora.yml", "configuration", "server", "rails_environment") ||
-      parse_value_from_file("defaults.yml", "defaults", "server", "rails_environment")
+      parse_value_from_toml_file("diaspora.toml", "rails_environment") ||
+      parse_value_from_yaml_file("diaspora.yml", "configuration", "server", "rails_environment") ||
+      parse_value_from_yaml_file("defaults.yml", "defaults", "server", "rails_environment")
   end
 
   def self.database
-    @adapter ||= parse_value_from_file("database.yml", rails_env, "adapter")
+    @adapter ||= parse_value_from_yaml_file("database.yml", rails_env, "adapter")
 
     raise "No database adapter found, please fix your config/database.yml!" unless @adapter
 
     @adapter.sub("mysql2", "mysql")
   end
 
-  private_class_method def self.parse_value_from_file(file, *keys)
+  private_class_method def self.parse_value_from_yaml_file(file, *keys)
     path = File.join(__dir__, file)
     return YAML.load_file(path).dig(*keys) if File.file?(path)
 
     puts "Configuration file #{path} not found, ensure it's present" # rubocop:disable Rails/Output
   end
+
+  private_class_method def self.parse_value_from_toml_file(file, key)
+    path = File.join(__dir__, file)
+    return File.read(path)[/^\s*#{Regexp.escape(key)}\s*=\s*["']([^"']+)["']\s*$/, 1] if File.file?(path)
+
+    puts "Configuration file #{path} not found, ensure it's present" # rubocop:disable Rails/Output
```

Comment:
I would move this warning also to the commit that goes to `develop` only, so it only starts warning after 0.8 is released, otherwise it could confuse podmins on a minor-upgrade (especially since we added `script/configure_bundler` to the upgrade guide now for #8131).

Also maybe add `WARNING:` in front of it, like with the other warning ;)

Reply:
Ah, well I copied it from the existing method :D Will do.